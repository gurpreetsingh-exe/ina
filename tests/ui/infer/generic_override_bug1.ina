// COMPILE_FAIL

extern mod core;
using core::prelude::*;

fn main() {
    let a = match Some![i64](20) {
        Some(v) => v,
        None => {},
    };
}
