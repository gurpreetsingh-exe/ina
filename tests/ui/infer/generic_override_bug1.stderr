error: mismatch types
   ╭─[tests/ui/infer/generic_override_bug1.ina:9:17]
   │
 9 │         None => {},
   │                 ╶╮
   │                  ╰── expected `i64`, found `()`
   ∙
───╯
